### PR TITLE
Display additional shares during scalar LPing

### PR DESCRIPTION
**Pull request by KadenZipfel**
Closes: https://github.com/protofire/omen-exchange/issues/1601

### CODE REVIEW

**Review by hexyls — APPROVED** [review]

**Review by pimato — APPROVED** [review]

File: app/src/components/market/common/market_scale/value_box.tsx [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+        ? `left: ${
+            props.xValue <= 0.115
+              ? `0`
+              : props.xValue <= 0.885
+              ? `${props.xValue * 100}%; transform: translateX(-50%);`
+              : ``
+          }`
+        : `right: 0;`
+      : ''}
+  background: white;
```
Comment (Mi-Lan) [inline app/src/components/market/common/market_scale/value_box.tsx:51]: should we maybe use value from theme here? and also closing column above it for the xValue...
Screenshot 2021-02-12 at 12 36 09

File: app/src/components/market/common/market_scale/value_box.tsx [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+  position: absolute;
+  top: calc(${SCALE_HEIGHT} + ${VALUE_BOXES_MARGIN});
+  border-radius: 4px;
+`
+
+const ValueBoxTitle = styled.p<{ positive?: boolean | undefined }>`
+  font-size: 14px;
```
Comment (Mi-Lan) [inline app/src/components/market/common/market_scale/value_box.tsx:58]: this value is also default font-size from theme but i would suggest adding just one font-size in wrapper valueBoxSegment so we don't have to repeat it for all the children...

File: app/src/components/market/common/market_scale/value_box.tsx [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+    border-bottom-right-radius: 4px;
+    border-top-left-radius: 0px;
+    border-bottom-left-radius: 0px;
+    border-left: none;
+  }
+`
+
+const ValueBoxSingle = styled.div<{ xValue?: number }>`
+  padding: 12px;
+  border: 1px solid ${props => props.theme.scale.box};
```
Comment (Mi-Lan) [inline app/src/components/market/common/market_scale/value_box.tsx:35]: also here i think we have borderLineDisabled in theme which has 1px solid with the desired color...

**Review by Mi-Lan — CHANGES_REQUESTED** [review]
Great job! Works like charm 🥇  Just added couple of small css things i found

**Review by Mi-Lan — APPROVED** [review]

**Pull request merged**